src/graphql/main/schema.graphql

```diff
@@ -730,9 +730,9 @@ type Mutation {
   lnUsdInvoiceFeeProbe(input: LnUsdInvoiceFeeProbeInput!): SatAmountPayload!
   onChainAddressCreate(input: OnChainAddressCreateInput!): OnChainAddressPayload!
   onChainAddressCurrent(input: OnChainAddressCurrentInput!): OnChainAddressPayload!
-  onChainPaymentSend(input: OnChainPaymentSendInput!): PaymentSendPayload!
+  onChainPaymentSend(input: OnChainPaymentSendInput!): PaymentSendPayload! // This should allow for usd wallets
```

Comment:
`onChainPaymentSend` needs to be able to accept a UsdWallet because to pay an onchain invoice with an amount, that amount will be denominated in sats, so we need a mutation that accepts a usd wallet while denominating the amount in sats.